Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectCardModel.swift

```diff
@@ -14,6 +15,18 @@ public struct PPOProjectCardModel: Identifiable, Equatable {
   public let tierType: TierType
   public let projectAnalytics: GraphAPI.ProjectAnalyticsFragment
 
+  public func hash(into hasher: inout Hasher) {
+    hasher.combine(self.isUnread)
+    hasher.combine(self.alerts)
+    hasher.combine(self.image)
+    hasher.combine(self.title)
+    hasher.combine(self.creatorName)
+    hasher.combine(self.address)
```

Comment:
nit: I'm a little confused by the choice to add the address to the hasher and not the pledge. I don't think we need either of them to guarantee uniqueness but I don't think it hurts to include them either. Is it just that the pledge isn't hashable already? Because I'd be fine with that reasoning!

Reply:
Added an implementation of Hashable for that fragment and added it to the hasher.